parm/config/config.eupdfsoi

```diff
@@ -0,0 +1,38 @@
+#!/bin/ksh -x
+
+########## config.eupdfsoi ##########
+# Ensemble update specific, dependency config.anal
+
+echo "BEGIN: config.eupdfsoi"
+
+# Get task specific resources
+. $EXPDIR/config.eupd
+
+# Get task specific resources
+. $EXPDIR/config.resources eupdfsoi
+
+export npe_enkf=$npe_eupd
+
+
+# Use NAM_ENKF below for serial EnKF
+export NAM_ENKF="analpertwtnh=0.9,analpertwtsh=0.9,analpertwttr=0.9"
```

Comment:
Same as above: put this in a control structure

Reply:
The layout of this file is a reflection of the existing parm/config/config.eupd . I can remove all of the non-serial lines since for the foreseeable future EFSOI will use only that